src/components/creation/UserGuidedCreation.react.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+
+import type {Node} from 'react';
+
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
```

Comment:
Pipeline builds are treated differently than local builds by default, so linting warnings are treated as errors.

Reply:
Thx for the feedback on that. That's pretty good